Library/Homebrew/manpages/brew.1.md

```diff
@@ -556,6 +556,11 @@ can take several different forms:
 
     *Note:* Homebrew will only try to print emoji on Lion or newer.
 
+  * HOMEBREW\_NO\_INSECURE\_REDIRECT:
+    If set, Homebrew will not permit redirects from secure download protocols
+    to insecure ones. Whist ensuring your downloads are fully secure, this can
+    have a major impact on Sourceforge, Apache & GNOME based formulae.
```

Comment:
I can be slightly more blunt I guess. `Will miserably break downloads of from-source Sourceforge, Apache & GNOME based formulae`? :wink: